packages/module/src/Ansible/Ansible.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import clsx from 'clsx';
 import { createUseStyles } from 'react-jss';
+import RHAAPTechnologyIcon from './rhaap-icon';
 
 export interface AnsibleProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> {
   /** Supported/unsupported variant flag */
   unsupported?: boolean | number;
+  /** Red Hat Anisble Automation Platform technology icon */
+  RHAAPIcon?: React.ReactNode
```

Comment:
```suggestion
  RHAAPIcon?: boolean
```
This one should probably be `boolean`.

I was also thinking how about renaming `unsupported` to `isUnsupported` and also `RHAAPIcon` to `isRHAAP`. Just for consistency and to make it more clear that it is a boolean flag.